src/System.Binary.Base64/System/Binary/Base64Decoder.cs

```diff
@@ -294,9 +211,10 @@ public TransformationStatus Transform(ReadOnlySpan<byte> source, Span<byte> dest
         /// <returns>Number of bytes written to the buffer.</returns>
         public static int DecodeInPlace(Span<byte> buffer)
         {
+            // TODO: Fix
```

Comment:
Definitely, and in general, I agree with the principle. However, I will make an effort to re-implement and resolve the TODO ASAP.

Otherwise, I will instead open an issue and detail what is required there. I prefer those over TODO comments within the code anyway.